### DISCUSSION TITLE
When will version 7.0.0 be released?

### DISCUSSION THREAD

**Discussion by robert94p** [rabbitmq/rabbitmq-dotnet-client#1569] [General]
When will version 7.0.0 be released? Is the version not stable yet?

**Comment by lukebakken** [comment]
> When will version 7.0.0 be released?

The goal is by the end of this month (May 2024).

If you would like to contribute, **PLEASE** test alpha releases as they are available:

https://www.nuget.org/packages/RabbitMQ.Client/7.0.0-alpha.5

Helping Team RabbitMQ by testing these releases is appreciated!

**Comment by lukebakken** [comment]
@robert94p please test this release in your environment:

https://github.com/rabbitmq/rabbitmq-dotnet-client/releases/tag/v7.0.0-alpha.6

Thanks!

**Comment by robert94p** [comment]
7.0.0 is not stable yet?

**Reply by lukebakken** [reply]
Really, there is no need to keep asking. It's obvious that version 7 is not yet officially released. The API for version 7 IS stable.

As I said earlier, testing release candidates would be greatly appreciated. Thank you.

https://www.nuget.org/packages/RabbitMQ.Client/#versions-body-tab

**Reply by michaelklishin** [reply]
@robert94p you can give it a shot, it is open source software. Asking others to do is for you is not how open source software works: you get it for free, so the least you can do is to help test it and reporting your findings.
